src/modules/auth/login.strategy.ts

```diff
@@ -2,27 +2,29 @@ import { LoginStrategy } from 'passport-did-auth';
 import { Inject, Injectable } from '@nestjs/common';
 import { PassportStrategy } from '@nestjs/passport';
 import { ConfigService } from '@nestjs/config';
+import { IpfsGatewayConfig, IPFSGatewayConfigToken } from '../ipfs/ipfs.types';
+import { LoginStrategyOptions } from 'passport-did-auth/dist/lib/LoginStrategy';
 
 @Injectable()
 export class AuthStrategy extends PassportStrategy(LoginStrategy, 'login') {
   constructor(
     configService: ConfigService,
-    @Inject('IPFSClientConfig') ipfsConfig
+    @Inject(IPFSGatewayConfigToken) ipfsConfig: IpfsGatewayConfig
   ) {
-    let loginStrategyParams = {
+    let loginStrategyParams: Omit<LoginStrategyOptions, 'jwtSecret'> = {
```

Comment:
`jwtSecret` is not used by LoginStrategy

Reply:
Do you know why this is?

Reply:
Sorry, I was not accurate. `jwtSecret` is required by LoginStrategy to encode logged in user, but we decided to login to ssi-hub without encoding and because of this don't passing jwtSecret